app/angular/src/client/preview/angular-beta/StorybookModule.ts

```diff
@@ -45,7 +44,7 @@ export const getStorybookModuleMetadata = (
   if (storyComponent) {
     deprecatedStoryComponentWarning();
   }
-  const component = storyComponent ?? parameters.component;
+  const component = storyComponent ?? annotatedComponent;
```

Comment:
this will be create braking change. i think 

because before, to have a specific component for a story, you had to pass it through the parameters

Now if I understand correctly we can make a : 

```
        export const Primary: Story = () => ({});
        Primary.component = AppComponent;
```

maybe add a message in the MIGRATION.md